### PR TITLE
python.pkgs.pybrowserid: move expression and  0.9.2 -> 0.14.0

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Some cleaning

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by worldofpeace** [comment]
cc @dotlambda ?

File: pkgs/development/python-modules/pybrowserid/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  version = "0.14.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qvi79kfb8x9kxkm5lw2mp42hm82cpps1xknmsb5ghkwx1lpc8kc";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  buildInputs = [ mock ];
```
Comment (xeji) [inline pkgs/development/python-modules/pybrowserid/default.nix]: Shouldn't this rather be `checkInputs` ?

Comment (worldofpeace) [reply pkgs/development/python-modules/pybrowserid/default.nix]: Ahh, yeah.

Comment (worldofpeace) [reply pkgs/development/python-modules/pybrowserid/default.nix]: Corrected.

**Comment by xeji** [comment]
@GrahamcOfBorg build python27Packages.pybrowserid python36Packages.pybrowserid  python27Packages.syncserver

**Pull request merged**